src/js/default-props.js

```diff
@@ -8,3 +8,11 @@ export const defaultProps = {
 export const extendDefaultTheme = theme => {
   defaultProps.theme = deepMerge(base, theme);
 };
+
+/*
+  Pass `theme` for component which can be located outside of theme context.
+  To be used as argument for `attrs` method from `styled-components`.
+ */
+export const enhancePropsWithTheme = props => ({
```

Comment:
just flagging if we want to chat here about naming

Reply:
Naming suggestions from grommet checkpoint:
* withTheme
* baseGrommetProps
* baseProps
* grommetProps
* baseGrommet

Reply:
went with `withTheme`